2x/docs/executable.html

```diff
@@ -205,7 +205,7 @@
 <h2>Options</h2>
 
 <pre><code>  -s, --sessions         Add session support
-  -t, --template ENGINE  Add template ENGINE support (jade|ejs). Defaults to jade
+  -t, --template ENGINE  Add template ENGINE support (pug|ejs). Defaults to pug
```

Comment:
This cannot be changed, because the actual `express` binary in 2.x hasn't been changed to `pug` and then the documentation would be incorrect.